packages/salesforcedx-vscode-lwc/test/vscode-integration/jsconfig.test.ts

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Position, window, workspace, WorkspaceEdit } from 'vscode';
+import URI from 'vscode-uri';
+
+const CONFIG_FILENAME = 'jsconfig.json';
+const TEST_COMPONENT_NAME = 'testComponent';
+const CREATE_COMPONENT_NAME = 'createComponent';
+
+describe('jsconfig Test Suite', () => {
+  let modulesDir: string;
+  let configPath: string;
+  let config: object;
+
+  before(async () => {
+    modulesDir = path.join(
+      workspace.workspaceFolders![0].uri.fsPath,
+      'force-app',
+      'main',
+      'default',
+      'lwc'
+    );
+    configPath = path.join(modulesDir, CONFIG_FILENAME);
+  });
+
+  beforeEach(async function() {
+    this.timeout(5000);
+
+    await createComponent(TEST_COMPONENT_NAME, modulesDir);
+    await waitForConfigUpdate(configPath);
+
+    config = await parseConfig(configPath);
+  });
+
+  it('Should update jsconfig.json on component creation', async function() {
```

Comment:
Hoping we can more accurately state what we're verifying.

it('adds the newly created component to the jsconfig compilerOptions paths map')